idea/src/org/jetbrains/kotlin/idea/intentions/ConvertLambdaLineIntention.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.psi.PsiComment
+import org.jetbrains.kotlin.idea.refactoring.getLineNumber
+import org.jetbrains.kotlin.psi.KtLambdaExpression
+import org.jetbrains.kotlin.psi.KtPsiFactory
+import org.jetbrains.kotlin.psi.psiUtil.allChildren
+
+sealed class ConvertLambdaLineIntention(private val toMultiLine: Boolean) : SelfTargetingIntention<KtLambdaExpression>(
+    KtLambdaExpression::class.java, "Convert to ${if (toMultiLine) "multi" else "single"}-line"
```

Comment:
Better add "lambda" (e.g,  "Convert to multi-line lambda") to the end of the title as it's not quite obvious for the user what should that intension do in case when many intentions are suggested to the user